src/news/release-1.3.0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Release 1.3.0
+description: A lot of new things to check out!
+date: 2022-05-22
+tags:
+  - release
+---
+
+Hey!
+It's update-time again. This time we bring you some highlights, like Modrinth modpack support, a Windows installer, more paste services and improvements around our ATLauncher support.
+
+Initially this release should have included a workaround for download issues with CurseForge mods and modpacks, but we were contacted by CurseForge to remove our workaround.
+
+## Changelog
+
+### Added
+- Support ATLauncher share codes by @jamierocks in #408
+- Allow changing metadata-server settings by @dada513 in #425
+- Add PolyMC icon as instance icon by @ryanccn in #498
+- Add option to always hide Java setup wizard by @dada513 in #511
+- Add support for paste.gg, hastebin and mclo.gs paste services by @LennyMcLennington in #554
+- Add instance overrides for miscellaneous settings by @kthchew in #560
+- Modrinth modpack support by @flowln and @Scrumplex in #565
+- Quilt mod metadata support by @Scrumplex in #567
+- Support Mojang's new split natives by @Scrumplex in #594
+- [macOS] Add Big Sur-style icon by @ryanccn in #495
+- [Windows] Add Windows installer by @kthchew in #576
+
+### Changed
+- Bump Qt requirement to 5.12 by @Scrumplex in #127
+- Refactor launcher code by @icelimetea in #486 #520
+- Refactor network code by @flowln in #500
+- Speedup automatic builds by @txtsd in #510 #614
+- Make profile selection menu scrollable by @kthchew in #546
+- Switch to new CurseForge API Key by @ZekeSmith in #557
+- Improve behavior when multiple mod loaders are in use by @Scrumplex in #597
+- Various improvements to ATLauncher support by @jamierocks in #605
+- Switch to a better CurseForge icon by @DioEgizio in #612
+- Improve compile parameters by @Scrumplex in #616 #620
+
+### Fixed
+- Always build iconfix as static library by @flowln in #477
+- Added and fixed mnemonics by @dschemp in #480 and @txtsd in #503
+- Fix selection of .zip files by @flowln in #528
+- Fix graphical issue when moving toolbars by @flowln in #529
+- Fix missing translations when switching languages by @kthchew in #539
+- Fix missing screenshots in screenshot manager by @kthchew in #540
+- Fix website link in CurseForge modpacks by @DioEgizio in #545
+- Fix toolbar disappearing in a certain circumstance by @LennyMcLennington in #577
+- Fix issue with Solder.cf Technic packs by @flowln in #592
+- [Windows] Hide temporary directory inside instances directory by @kthchew in #547
```

Comment:
This applies to macOS and Linux too, by the way (since they usually hide files starting with a dot by default). The Windows specific stuff was just because the dot trick doesn't hide files there
```suggestion
- Hide temporary directory inside instances directory by @kthchew in #547
```

Reply:
Right. I overlooked that part. I only saw Windows header includes and instantly wanted to close my browser tab again 😅

Reply:
That's a fair response for Windows to cause